.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,36 @@
 ## Definition of Done
 
-**General**
- - [ ] Your branch is up-to-date with development branch
- - [ ] A `Readme.md` file is generated as part of the blueprint which, at the minimum, explains how to use the blueprint after it is generated(like adding any missing steps, creating accounts, setting up docker containers, applying the yaml using XL-CLI, running release etc). Place this file preferably in the XebiaLabs folder. Do not add the blueprint generation steps here as it wont be useful since the file is ceated only after the blueprint generation.
- - [ ] The blueprint must use an XL product at some capacity
- - [ ] There are no sensitive info in blueprints(tokens, credentials, licence), for questions that ask for password or sensitive info use `secret: true` in the `blueprint.yaml` parameter definition.
- - [ ] Do not add the `xebialabs/.gitignore`, `xebialabs/values.xlvals` and `xebialabs/secrets.xlvals` to the `blueprint.yaml` files section, and do not check in those files, as they are autogenerated based on parameter definitions. Secrets are created when a parameter in the `blueprint.yaml` has the `secret: true` directive. For adding parameters to `values.xlvals` use the `saveInXlVals: true` directive.
- - [ ] Please do not ask questions for trivial things like phase names, task names, folder names etc, ask questions that add value and be opinionated where possible. For example ask for a project name and derive folder names, task names etc based on that.
- - [ ] Folder and file names must use dash-case, for example: `aws/sample-app-demo`, `xld-environment.yaml`
- 
+This is the definition of done for (new and modified) blueprints to be accepted into the [central XebiaLabs blueprints repository](https://github.com/xebialabs/blueprints).
+
+**Value**
+
+This section describes how to determine whether a blueprint has enough value to be included in the central XebiaLabs blueprints repository.
+
+- [ ] The blueprint applies to a focused use case.
+- [ ] The blueprint uses at least one XebiaLabs product.
+- [ ] The blueprint depends on as few other products as possible. **N.B.:** This is part of the definition of done to keep the blueprint focused and applicable to as many users as possible. A blueprint that depends on one spefic CI tool, one specific Java EE application server and one specific ticketing system can only be used by users that have all three products.
+- [ ] The blueprint can be used with content supplied by the user but also includes demo content. A blueprint that does not work with content supplied by the user is a demo or a showcase and does not help the user with their own work.
```

Comment:
@vpartington but we already have showcases right, like data-lake, this would kind of mean that we only will have starter/generic ones.

Reply:
may be we say, demo/showcase blueprints will only be accepted on a case to case basis

Reply:
True. I was thinking of the data lake blueprint when I wrote that line. To be honest, I don't think that should really be a blueprint.

But indeed, maybe the solution for now it to say that demo/showcase blueprints will be accepted on a case-by-case basis.

In general, I'd say that a DoD tried to be as strict as possible. But that exceptions are always possible.